README.md

```diff
@@ -51,6 +51,10 @@ to ``ovirt_engine_setup_answer_file_path`` variable.
 | ovirt_engine_setup_perform_upgrade    | False                 | If `True`, this role is used to perform an upgrade. |
 | ovirt_engine_setup_product_type       | oVirt                 | One of ["oVirt", "RHV"], case insensitive. |
 | ovirt_engine_setup_offline            | False                 | If `True`, updates for all packages will be disabled. |
+| ovirt_engine_setup_restore_engine_cleanup | False             | Remove the configuration files and clean the database associated with the Engine, relevant only when `ovirt_engine_setup_restore_file` is defined |
+| ovirt_engine_setup_restore_file       | UNDEF                 | Restored the engine with a backup file which created with engine-backup. |
+| ovirt_engine_setup_restore_scopes     | UNDEF                 | List of scopes following values are available: `all`, `files`, `db`, `dwhdb`, `cinderlibdb`. |
```

Comment:
just nitpick add square brackets around all options so it is like `ovirt_engine_setup_product_type`

Reply:
Done